### ISSUE TITLE
parser_missing_space - false positive

### ISSUE THREAD

**Issue by DGRHicron** [abaplint/abaplint#3761]
False positive for rule: parser_missing_space

```abap
REPORT zfoo.
DATA positions TYPE STANDARD TABLE OF string WITH KEY table_line.
DATA material TYPE matnr VALUE 'TEST'.

TYPES:
  BEGIN OF ty_line,
    material TYPE matnr,
  END OF ty_line.

START-OF-SELECTION.

  FIELD-SYMBOLS <position> TYPE ty_line.
  xsdbool( line_exists( positions[ ('MATERIAL') = <position>-material ] ) ).
  DATA(foo) = xsdbool( line_exists( positions[ ('MATERIAL') = <position>-material ] ) ).
  WRITE foo.
```

<img width="824" height="437" alt="Image" src="https://github.com/user-attachments/assets/1852e185-5e89-4709-91bc-fb0acc81b86b" />

**Comment by larshp** [comment]
thanks, reproduced
